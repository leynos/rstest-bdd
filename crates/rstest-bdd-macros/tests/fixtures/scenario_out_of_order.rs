use rstest_bdd_macros::{given, scenario};

#[scenario(path = "../../../../crates/rstest-bdd-macros/tests/features/ambiguous.feature")]
fn out_of_order() {}

#[given("a step")]
fn a_step() {}

fn main() {}
