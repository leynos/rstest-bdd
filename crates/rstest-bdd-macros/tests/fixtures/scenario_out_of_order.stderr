warning: Step registry does not contain definitions for crate_id '$CRATE:'. This may indicate a registry issue.
 --> tests/fixtures/scenario_out_of_order.rs:3:1
  |
3 | #[scenario(path = "../../../../crates/rstest-bdd-macros/tests/features/ambiguous.feature")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this warning originates in the attribute macro `scenario` (in Nightly builds, run with -Z macro-backtrace for more info)

error: No matching step definition found for: Given a step
 --> tests/fixtures/scenario_out_of_order.rs:3:1
  |
3 | #[scenario(path = "../../../../crates/rstest-bdd-macros/tests/features/ambiguous.feature")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `scenario` (in Nightly builds, run with -Z macro-backtrace for more info)
